intrahost.py

```diff
@@ -580,7 +595,10 @@ def merge_to_vcf(
                         alignmentFile)
 
                 for s in samplesToUse:
-                    for row in util.file.read_tabfile(samp_to_isnv[sampleIDMatch(s)]):
+                    isnv_filepath = samp_to_isnv[sampleIDMatch(s)]
+                    if os.path.getsize(isnv_filepath) == 0:
```

Comment:
Why the explicit check for `os.path.getsize() == 0`? Are there special scenarios where we pass in empty files that semantically mean something we should skip, or is this just a normal situation where vphaser detects no isnvs for that particular file (and thus produces empty output)? `util.file.read_tabfile` should simply produce an empty iterator, so this check seems unnecessary?

Reply:
You're right, this should not be necessary.